### PR TITLE
feat(UIForm): radios

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
**Warning: ** this is only available for the next version of forms
Radios are not supported

**What is the chosen solution to this problem?**
Impl Radios widgets
Improve accessibility by marking the focused radio

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/forms/src/UIForm/fields/Radios.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		isValid: PropTypes.bool,
+		errorMessage: PropTypes.string,
+		onChange: PropTypes.func,
+		schema: PropTypes.shape({
+			autoFocus: PropTypes.bool,
+			description: PropTypes.string,
+			disabled: PropTypes.bool,
+			inline: PropTypes.bool,
+			title: PropTypes.string,
+			type: PropTypes.string,
```
Comment (charlesnguyen) [inline packages/forms/src/UIForm/fields/Radios.js]: where is it used?

File: packages/forms/src/UIForm/fields/Radios.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import classNames from 'classnames';
+import FieldTemplate from './FieldTemplate';
+
+export default function Radios(props) {
+	const { id, isValid, errorMessage, onChange, schema, value } = props;
+	const { autoFocus, description, disabled, inline, title } = schema;
+
+	const radioClassNames = classNames({
+		radio: !inline,
+		'radio-inline': inline,
```
Comment (charlesnguyen) [inline packages/forms/src/UIForm/fields/Radios.js]: For inline case, we do NOT have the pointer cursor. Tested on Chrome and FF

Comment (jsomsanith) [inline packages/forms/src/UIForm/fields/Radios.js]: Done

File: packages/forms/src/UIForm/fields/Radios.js (hunk 1 above)
Comment (jsomsanith) [inline packages/forms/src/UIForm/fields/Radios.js]: Removed

File: packages/forms/src/UIForm/fields/Radios.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import FieldTemplate from './FieldTemplate';
+
+export default function Radios(props) {
+	const { id, isValid, errorMessage, onChange, schema, value } = props;
```
Comment (talend-sromain) [inline packages/forms/src/UIForm/fields/Radios.js]: Move the spread to the function call ?

File: packages/forms/src/UIForm/fields/Radios.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	});
+	return (
+		<FieldTemplate
+			description={description}
+			errorMessage={errorMessage}
+			isValid={isValid}
+			label={title}
+		>
+			{
+				schema.titleMap && schema.titleMap.map((option, index) => (
```
Comment (talend-sromain) [inline packages/forms/src/UIForm/fields/Radios.js]: How about move the map renderer to a separated function ?

**Pull request merged**